generator/config.js

```diff
@@ -12,7 +12,8 @@ const defaultConfig = {
   modelsOnly: false,
   outDir: "src/models",
   roots: [],
-  noReact: false
+  noReact: false,
+  namingConvention: "asis" // supported option: "js"
```

Comment:
My opinion would be to default this to `js`

Reply:
I just pushed my changes with --dontRenameModels and defaulting namingConvention to "js".